.gitignore

```diff
@@ -1,3 +1,6 @@
+# Visual Studio cache/options directory
+.vs/
```

Comment:
Added option to ignore Visual Studio options directory, since the edits to the repo are made using Visual Studio.